src/CONST/index.js

```diff
@@ -1,6 +1,7 @@
 import lodashGet from 'lodash/get';
```

Comment:
Why did the name of this file change? I don't think it should be changed. The only time we should ever have an `index.js` is when it's in a directory used to load platform specific files (eg. `index.js`, `index.website.js`, `index.ios.js`)